### ISSUE TITLE
PacketPlayOutEntityTeleport Error

### ISSUE THREAD

**Issue by WolverinDEV** [WolverinDEV/BungeeUtil#44]
BungeeUtil: 2.5.4
BungeeCord: 1250
Client: 1.12
Server: 1.9.4
Viaversion: 1.1.1

StackTrace:
```
00:10:23 [SEVERE] java.lang.RuntimeException: Packet error (Version: v1_9_3, Class: null, Id: 0x4A) -> Did not read all bytes from packet (dev.wolveringer.bungeeutil.packets.PacketPlayOutEntityTeleport)
00:10:23 [SEVERE] 	at dev.wolveringer.bungeeutil.packets.Packet.load(Packet.java:107)
00:10:23 [SEVERE] 	at dev.wolveringer.bungeeutil.packets.creator.NormalPacketCreator.getPacket0(NormalPacketCreator.java:90)
00:10:23 [SEVERE] 	at dev.wolveringer.bungeeutil.packets.creator.NormalPacketCreator.getPacket0(NormalPacketCreator.java:102)
00:10:23 [SEVERE] 	at dev.wolveringer.bungeeutil.packets.creator.NormalPacketCreator.getPacket0(NormalPacketCreator.java:109)
00:10:23 [SEVERE] 	at dev.wolveringer.bungeeutil.packets.creator.CachedPacketCreator.getPacket0(CachedPacketCreator.java:134)
00:10:23 [SEVERE] 	at dev.wolveringer.bungeeutil.packets.creator.AbstractPacketCreator.getPacket(AbstractPacketCreator.java:51)
00:10:23 [SEVERE] 	at dev.wolveringer.bungeeutil.packetlib.PacketRegistry.getPacket(PacketRegistry.java:131)
00:10:23 [SEVERE] 	at dev.wolveringer.bungeeutil.netty.WarpedMinecraftDecoder.decode(WarpedMinecraftDecoder.java:98)
00:10:23 [SEVERE] 	at dev.wolveringer.bungeeutil.netty.WarpedMinecraftDecoder.decode(WarpedMinecraftDecoder.java:32)
00:10:23 [SEVERE] 	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:88)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
00:10:23 [SEVERE] 	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
00:10:23 [SEVERE] 	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:310)
00:10:23 [SEVERE] 	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:297)
00:10:23 [SEVERE] 	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:413)
00:10:23 [SEVERE] 	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
00:10:23 [SEVERE] 	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
00:10:23 [SEVERE] 	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1334)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
00:10:23 [SEVERE] 	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
00:10:23 [SEVERE] 	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:926)
00:10:23 [SEVERE] 	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:979)
00:10:23 [SEVERE] 	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe$1.run(AbstractEpollChannel.java:338)
00:10:23 [SEVERE] 	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
00:10:23 [SEVERE] 	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:403)
00:10:23 [SEVERE] 	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:311)
00:10:23 [SEVERE] 	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858)
00:10:23 [SEVERE] 	at java.lang.Thread.run(Thread.java:745)
```

**Comment by WolverinDEV** [comment]
@essem2 
Hey,
i moved you error to an new issue.

Can you enable the "debug" mode and the it should print the packet contains if a packet deserialization fails. Maybe this could help me out :)

**Comment by kaerns** [comment]
everything from connection to leave: https://hastebin.com/oxoneqoked.md

**Comment by WolverinDEV** [comment]
@essem2 
Thank you very mutch :)
Im not verry sure that im failed my Packet ID mapping....
You're able to see in the packet dump some strings, this is a big indicator for the PacketPlayOUtEntityPropertyPacket with is just above the EntityTeleport packet.
(0x49 and 0x4a)

**Comment by kaerns** [comment]
Tested Today with
BungeeUtil: 2.5.5
BungeeCord: 1258
Client: 1.9.4, 1.10.2, 1.11.2, 1.12.1
Server: 1.9.4
Viaversion: 1.2.0
and no errors at all!

Offtopic: a typo in Attention here: https://github.com/WolverinDEV/BungeeUtil/blob/2.0/src/main/java/dev/wolveringer/bungeeutil/plugin/Main.java#L41

**Comment by WolverinDEV** [comment]
Hmm means that that this error is fixed? LOL :D 

Oh fuck i just saw it thanks man ;)

**Comment by WolverinDEV** [comment]
Seems to be fixed #51
